src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/webservice.yml

```diff
@@ -0,0 +1,6 @@
+admin_webservice:
+    path: /
+    methods: [GET, POST]
```

Comment:
here, what don't you use a new route for POST who go directly on `Webservice:process` action?

Reply:
Just for this form it works. However in the next step I will add the Grid to this page. In the previous implementation of the grid, the Search would send POST requests to the page so it requires this route to enable POST.

Did this change ?